docs/RUNNING-LOCALLY.md

```diff
@@ -21,3 +23,6 @@ cd ../
 ./build.ps1 build
 ./build.ps1 run
 ```
+
+## Running the EdFi.DataManagementService.Api.Tests.Integration
+> Note: If using a custom connection string for your integration tests, it's highly recommended to add an `appsettings.Test.json` file for your local development environment testing. This file should be based on the `appsettings.json` file on EdFi.DataManagementService.Api.Tests.Integration folder.
```

Comment:
Please wrap long lines. See [Markdown best practices](https://edfi.atlassian.net/wiki/spaces/ETKB/pages/20875522/Markdown+Best+Practices) for VS Code extensions, if needed.